### ISSUE TITLE
Error trying to give multiple licenses

### ISSUE THREAD

**Issue by talex5** [ocaml/dune#6103]
## Expected Behavior

https://dune.readthedocs.io/en/latest/dune-files.html?highlight=license#generate-opam-files-1 says:

> `(license <names>)` - specifies the license of the project, ideally as an identifier from the [SPDX License List](https://spdx.org/licenses/). Multiple licenses may be specified.

## Actual Behavior

I tried:

```
(license "ISC" "MIT")   # Error: Too many argument for license
```

```
(license ("ISC" "MIT"))  # Error: Atom or quoted string expected
```

```
(license "ISC")
(license "MIT")   # Error: Field "license" is present too many times
```

The following is accepted by dune, but is rejected by opam-repository (see https://github.com/ocaml/opam-repository/pull/18944/commits/d3ea851e9730368a1d74ebb18cf3a021922e34ac).

```
(license "(ISC AND MIT)")
```

It is possible to work around this using `PKG.opam.template` files.

## Specifications

```
$ dune --version
3.4.1

$ ocamlc --version
5.0.0~alpha1
```

**Comment by nojb** [comment]
Did you try `(license ISC MIT)` ?

**Comment by nojb** [comment]
(This was fixed in #5579)

**Comment by talex5** [comment]
Ah, it needs `(lang dune 3.2)`!

Could do with a better error message, perhaps.
